src/components/UI/Anticheat/index.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useEffect, useState } from 'react'
+import { AntiCheatInfo, GameInfo } from 'src/types'
+import { createNewWindow, ipcRenderer } from 'src/helpers'
+
+import './index.css'
+import { useTranslation } from 'react-i18next'
+
+type Props = {
+  gameInfo: GameInfo
+}
+
+const awacyUrl = 'https://areweanticheatyet.com/'
+
+export default function Anticheat({ gameInfo }: Props) {
+  const { t } = useTranslation()
+
+  const [anticheatInfo, setAnticheatInfo] = useState<AntiCheatInfo | null>(null)
+
+  useEffect(() => {
+    if (gameInfo?.title) {
+      ipcRenderer
+        .invoke('getAnticheatInfo', gameInfo.title)
+        .then((anticheatInfo: AntiCheatInfo | null) => {
+          setAnticheatInfo(anticheatInfo)
+        })
+    }
+  }, [gameInfo])
+
+  if (!anticheatInfo) {
+    return null
+  }
+
+  const mayNotWork = ['Denied', 'Broken'].includes(anticheatInfo.status)
+
+  return (
+    <div className={`anticheatInfo ${anticheatInfo.status}`}>
+      <h4>{t('anticheat.title', 'This game includes anticheat software')}</h4>
+      {mayNotWork && (
+        <p>
+          {t(
+            'anticheat.may_not_work',
+            'It may not work due to denied or broken anticheat support.'
+          )}
+        </p>
+      )}
+      <span>
+        <b>{t('anticheat.anticheats', 'Anticheats')}:</b>&nbsp;
+        {anticheatInfo.anticheats.length
```

Comment:
I wonder if `anticheats` can be something else other than an array or string, this could break.
Maybe do `anticheatInfo.anticheats?.length`

Reply:
I could add that just to be safe (doesn't hurt), but they do `anticheats.map` directly in the website without a guard https://github.com/Starz0r/AreWeAntiCheatYet/blob/master/components/GamesList.tsx#L85 and in the games.json I only saw arrays of strings (or empty array in the case of Far Cry 5)